webdriver/tests/support/fixtures_bidi.py

```diff
@@ -37,3 +38,19 @@ async def on_event(method, data):
 
         return future
     return wait_for_event
+
+@pytest.fixture
+def current_time_bidi(bidi_session, top_context):
```

Comment:
Oh we completely missed that the current_time was using current_session.
We actually only use the fixture for bidi tests.

Let's remove the one from fixtures_http.py and keep the current_time name.
We will only have to add the await in the call sites.

Reply:
Done